examples/bri-3/prisma/migrations/20221027145638_add_transaction_bpi_accounts_fks/migration.sql

```diff
@@ -0,0 +1,20 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `from` on the `Transaction` table. All the data in the column will be lost.
+  - You are about to drop the column `to` on the `Transaction` table. All the data in the column will be lost.
+  - Added the required column `fromBpiAccountId` to the `Transaction` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `toBpiAccountId` to the `Transaction` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "Transaction" DROP COLUMN "from",
+DROP COLUMN "to",
+ADD COLUMN     "fromBpiAccountId" TEXT NOT NULL,
```

Comment:
@Manik-Jain should this not be `toBpiSubjectAccount` and `fromBpiSubjectAccount`? since the transaction is between BPI subjects not BPI accounts since a transaction is impacting the state of a BPI account

Reply:
After looking at the standard one more time[ R257](https://github.com/eea-oasis/baseline-standard/blob/main/core/baseline-core-v1.0-psd01.md#r257) and [R258](https://github.com/eea-oasis/baseline-standard/blob/main/core/baseline-core-v1.0-psd01.md#r258) are not precise. They should read as follows:
[R257]
A Transaction MUST have a From (Sender) and a To (Receiver) element each containing the respective Sender and Receiver BPI Subject Account numbers.

[R258]
Each transaction MUST have a deterministic nonce from the BPI Account that is attached to the Workstep Instance ID. cc @Ybittan

Reply:
@skosito  can you please check this one?